src/app/test/dot-themes-service.mock.ts

```diff
@@ -0,0 +1,9 @@
+import { DotTheme } from '../portlets/dot-edit-page/shared/models/dot-theme.model';
+import { mockDotThemes } from './dot-themes.mock';
+import { Observable } from 'rxjs/Observable';
+
+export class DotThemesServiceMock {
+    get(inode: string): Observable<DotTheme[]> {
+        return Observable.of(mockDotThemes);
```

Comment:
it's better idea return a copy of mockDotThemes here, I mean something like:

Object.merge({}, mockDotThemes)